core/src/test/java/org/elasticsearch/search/aggregations/metrics/ExtendedStatsTests.java

```diff
@@ -32,4 +38,15 @@ protected ExtendedStatsAggregationBuilder doCreateTestAggregatorFactory() {
         return factory;
     }
 
+    @Test
```

Comment:
Our build does not allow the use of the `@Test` annotation and instead we have the convention naming test methods starting with `test`. Could you change this test method to have a name beginning with test? Also it might be a good idea to run `gradle precommit` after making the change to find out if there are any other issues.